k8s/core/configmap/configmap_lock_v2.go

```diff
@@ -387,20 +387,27 @@ func (c *configMap) refreshLock(id, key string) {
 		select {
 		case <-refresh.C:
 			lock.Lock()
-
+			isConflictErrCount := 0
 			for !lock.unlocked {
 				c.checkLockTimeout(c.defaultLockHoldTimeout, startTime, id)
 				currentRefresh = time.Now()
 				if _, err := c.tryLock(id, key, true); err != nil {
+					if k8s_errors.IsConflict(err) {
+						isConflictErrCount++
+						if isConflictErrCount%10 == 0 {
+							configMapLog(fn, c.name, "", key, err).Errorf(
+								"Error refreshing lock due to conflict from concurrent configmap updates. retries: %v."+
+									" [ID %v] [Key %v] [Err: %v] [Current Refresh: %v] [Previous Refresh: %v]",
+								isConflictErrCount, id, key, err, currentRefresh, prevRefresh)
+						}
+						// try refreshing again
+						continue
```

Comment:
I am thinking..it is possible that multiple nodes are stuck in this same state then they will continuously get conflict errors and we continuing from here will not help either. That is the current behavior too but we can improve this a little by adding a sleep (1sec + random jitter).

Reply:
Makes a lot of sense